adapters/primitive-adapters/api/primitive-adapters.klib.api

```diff
@@ -0,0 +1,22 @@
+// Klib ABI Dump
+// Targets: [androidNativeArm32, androidNativeArm64, androidNativeX64, androidNativeX86, iosArm64, iosSimulatorArm64, iosX64, js, linuxArm64, linuxX64, macosArm64, macosX64, mingwX64, tvosArm64, tvosSimulatorArm64, tvosX64, watchosArm32, watchosArm64, watchosDeviceArm64, watchosSimulatorArm64, watchosX64]
+// Rendering settings:
+// - Signature version: 2
+// - Show manifest properties: true
+// - Show declarations: true
+
+// Library unique name: <app.cash.sqldelight:primitive-adapters>
```

Comment:
@JakeWharton so you mean this line that should be `// Library unique name: <app.cash.sqldelight:sqldelight-primitive-adapters>`?